apps/admin/frontend/src/api.ts

```diff
@@ -316,53 +318,54 @@ export const getWriteInCandidates = {
   },
 } as const;
 
-type GetCvrWriteInImageViewsInput = QueryInput<'getWriteInImageViews'>;
-export const getCvrWriteInImageViews = {
-  queryKey(input?: GetCvrWriteInImageViewsInput): QueryKey {
-    return input
-      ? ['getCvrWriteInImageViews', input]
-      : ['getCvrWriteInImageViews'];
+type GetWriteInsInput = QueryInput<'getWriteIns'>;
+export const getWriteIns = {
+  queryKey(input?: GetWriteInsInput): QueryKey {
+    return input ? ['getWriteIns', input] : ['getWriteIns'];
   },
-  useQuery(input?: GetCvrWriteInImageViewsInput) {
+  useQuery(input?: GetWriteInsInput) {
     const apiClient = useApiClient();
     return useQuery(
       this.queryKey(input),
       () =>
-        apiClient.getWriteInImageViews({
-          cvrId: input?.cvrId ?? '',
-          contestId: input?.contestId ?? '',
+        apiClient.getWriteIns({
+          cvrId: input?.cvrId,
+          contestId: input?.contestId,
         }),
       { enabled: !!input, keepPreviousData: true }
     );
   },
-  usePrefetch() {
-    const apiClient = useApiClient();
-    const queryClient = useQueryClient();
-    return (input: GetCvrWriteInImageViewsInput) =>
-      queryClient.prefetchQuery({
-        queryKey: getCvrWriteInImageViews.queryKey(input),
-        queryFn: () => apiClient.getWriteInImageViews(input),
-      });
-  },
 } as const;
 
-type GetWriteInsInput = QueryInput<'getWriteIns'>;
-export const getWriteIns = {
-  queryKey(input?: GetWriteInsInput): QueryKey {
-    return input ? ['getWriteIns', input] : ['getWriteIns'];
+type GetWriteInImageViewsInput = QueryInput<'getWriteInImageViews'>;
+export const getWriteInImageViews = {
+  queryKey(input?: GetWriteInImageViewsInput): QueryKey {
+    return input ? ['getWriteInImageViews', input] : ['getWriteInImageViews'];
   },
-  useQuery(input?: GetWriteInsInput) {
+  useQuery(input?: GetWriteInImageViewsInput) {
     const apiClient = useApiClient();
     return useQuery(
       this.queryKey(input),
-      () =>
-        apiClient.getWriteIns({
-          cvrId: input?.cvrId,
-          contestId: input?.contestId,
-        }),
+      input
+        ? () =>
+            apiClient.getWriteInImageViews({
+              cvrId: input.cvrId,
+              contestId: input.contestId,
+            })
+        : /* istanbul ignore next - @preserve */
+          () => fail('input is required'),
```

Comment:
I've added this for two more of the queries that use this `enabled: !!input` pattern